apps/src/StudioApp.js

```diff
@@ -2769,6 +2769,7 @@ StudioApp.prototype.setPageConstants = function (config, appSpecificConstants) {
     isShareView: !!config.share,
     pinWorkspaceToBottom: !!config.pinWorkspaceToBottom,
     instructionsInTopPane: !!config.showInstructionsInTopPane,
+    hasContainedLevels: config.hasContainedLevels,
```

Comment:
Can this be `config.containedLevelOps.length > 0` to avoid passing down another view option?